presto-bigquery/src/main/java/io/prestosql/plugin/bigquery/BigQueryCredentialsSupplier.java

```diff
@@ -29,14 +29,10 @@
 
 class BigQueryCredentialsSupplier
 {
-    private final Optional<String> credentialsKey;
-    private final Optional<String> credentialsFile;
     private final Supplier<Optional<Credentials>> credentialsCreator;
 
     public BigQueryCredentialsSupplier(Optional<String> credentialsKey, Optional<String> credentialsFile)
     {
-        this.credentialsKey = credentialsKey;
-        this.credentialsFile = credentialsFile;
```

Comment:
good that you removed unused fields

we should still validate the values are non null, as they are not validated otherwise (even implicitly) before the ctor returns